### DISCUSSION TITLE
Collector not receiving logs

### DISCUSSION THREAD

**Discussion by Gschragg** [cisagov/LME#144] [FAQ] [answered]
I've already made it to chapter 2 of the setup process and was about to move onto Chapter 3, before I was able to get to chapter 2 however I ran into a problem with the collector and the client where it wasn't recognizing the client I have been using for testing. I was given the following error code in event viewer>Eventlog-ForwardingPlugin. 
![image](https://github.com/cisagov/LME/assets/156207169/351a698f-8677-4ca0-aa53-1b132b9a74a9)
The WEC also wasn't receiving a connection from the client, after a days worth of troubleshooting found out COM+ needs to be allowed on the WEC for it to receive that connection. I'm still given that error but at least the client is recognized on the WEC. 

So I went onto Chapter 2 hoping maybe the problem will work itself out, now that I've completed chapter 2 I'm having a problem where the client isn't sending over any logs, the task scheduled to run update.bat does work and sysmon64.exe is running as a service on the cilent, but when I visit sysmon on event viewer I am prompted with  
![image](https://github.com/cisagov/LME/assets/156207169/591baa2b-f0f1-4413-ac9b-d0d69ce6a524)
which seems like a very generic error, at this point I'm holding off from going to chapter 3 until I can resolve 1 if not both problems. 

I have double checked all steps and made sure all GPOs are configured correctly, any help would be much appreciated, I'm thinking I might have to nuke the WEC server and start fresh but I'll hold off for a couple days till I can get an answer from anyone.

Thank you in advanced!

**Comment by llwaterhouse** [comment, marked as answer]
Hello, thank you for your inquiry.  In order to better help you, please resubmit this ticket as a [githubIssue](https://github.com/cisagov/LME/issues),  with the fields of the template filled in.
